pkg/precompiles/warp.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package precompiles
+
+import (
+	_ "embed"
+	"fmt"
+
+	"github.com/ava-labs/avalanche-cli/pkg/contract"
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+func WarpPrecompileGetBlockchainID(
```

Comment:
Do we have any sort of framework/etc in place for testing functions like these (ie one's that would require a lot of mocking)?

Reply:
we can test this on local network. Do u want to create separate PR for this? @felipemadero

Reply:
we don't have framework in place for testing this as unit test. we usually do E2E testing with a local network
deployment, not with a mock. for sure we can consider mocking if we want to really add testing for 
all code paths.

Reply:
really, how to unit test this stuff, and how to properly tests the commands ux, for example
(prompts, flags paths), needs good design

Reply:
open to freely talk on implementation options

Reply:
regarding testing this specific feature on E2E, we don't have a command to return blockchain ID,
but it is going to be part of a larger feature in the short term. 
https://github.com/ava-labs/avalanche-cli/issues/2637

Reply:
do we want to add annotations to the functions to point to the specific E2E?